static/src/js/containers/DatepickerContainer/DatepickerContainer.jsx

```diff
@@ -140,7 +172,7 @@ class DatepickerContainer extends Component {
     // if so, we convert it to a UTC string in our desired format. If the value is not a valid ISO date,
     // then we leave it untouched and pass it to the input.
     // We are using YYYY-MM-DDTHH:m:s.SSSZ instead of moment.ISO_8601 so that it doesn't autocomplete every time there's a valid ISO format
-    const isValidISO = moment.utc(value, 'YYYY-MM-DDTHH:mm:ss.SSSZ', true).isValid()
+    const isValidISO = moment.utc(value, moment.ISO_8601, true).isValid()
```

Comment:
Why is the one different than the others also the comment above we should fix if this is needed `'YYYY-MM-DDTHH:mm:ss.SSSZ'` was used everywhere else